server/src/main/java/org/opensearch/index/IndexSettings.java

```diff
@@ -525,6 +536,8 @@ public IndexSettings(final IndexMetadata indexMetadata, final Settings nodeSetti
         mergeSchedulerConfig = new MergeSchedulerConfig(this);
         gcDeletesInMillis = scopedSettings.get(INDEX_GC_DELETES_SETTING).getMillis();
         softDeleteEnabled = version.onOrAfter(LegacyESVersion.V_6_5_0) && scopedSettings.get(INDEX_SOFT_DELETES_SETTING);
+        translogPruningByRetentionLease = version.onOrAfter(LegacyESVersion.V_6_5_0) &&
```

Comment:
This code will be pushed in Opensearch version. How would adding 6.5 version here help? Maybe add respective versions when you backport this change?

Reply:
Ack